commons/src/main/java/org/infinispan/commons/api/AsyncCache.java

```diff
@@ -315,11 +315,58 @@
       });
    }
 
+   /**
+    * Asynchronous version of {@link BasicCache#compute(Object, BiFunction)}. This method does not block on remote
+    * calls, even if your cache mode is synchronous. Has no benefit over {@link BasicCache#compute(Object, BiFunction)}
+    * )} if used in LOCAL mode.
```

Comment:
I only noticed this today, but this is no longer true. @pruivo modified the locking interceptors to not block while acquiring the lock, so there's a difference even in a local cache, we should update the javadoc both for the new methods and for the existing ones.